Sources/tart/Platform/Darwin.swift

```diff
@@ -98,11 +98,17 @@ struct Darwin: Platform {
     return result
   }
 
+  func keyboards() -> [VZKeyboardConfiguration] {
+    if #available(macOS 14, *) {
+      return [VZMacKeyboardConfiguration()]
+    } else {
+      return [VZUSBKeyboardConfiguration()]
+    }
+  }
+
   func pointingDevices() -> [VZPointingDeviceConfiguration] {
     if #available(macOS 13, *) {
-      // Trackpad is only supported starting with macOS Ventura
-      // macOS Monterey will continue using a USB device == .darwin 
-      return [VZMacTrackpadConfiguration(), VZUSBScreenCoordinatePointingDeviceConfiguration()]
```

Comment:
Will it make Monterey guests not having any pointing devices?

Reply:
As far as I understood from [`VZMacTrackpadConfiguration` documentation](https://developer.apple.com/documentation/virtualization/vzmactrackpadconfiguration), the underlying device is selected automatically:

>In macOS 13 and later, guests use the multi-touch trackpad device, while earlier versions of macOS uses the USB pointing device.

I've tried running this change with a Monterey guest, and it works.